Sources/Plasma/FeatureLib/pfPython/cyMiscGlue4.cpp

```diff
@@ -695,66 +695,24 @@ PYTHON_GLOBAL_METHOD_DEFINITION(PtSetBehaviorNetFlags, args, "Params: behKey, ne
 
 PYTHON_GLOBAL_METHOD_DEFINITION(PtSendFriendInvite, args, "Params: emailAddress, toName = \"Friend\"\nSends an email with invite code")
 {
-    PyObject* emailObj;
-    PyObject* toNameObj = nil;
-    if (!PyArg_ParseTuple(args, "O|O", &emailObj, &toNameObj))
+    char* emailIn = nullptr;
+    char* nameIn = nullptr;
+    if (!PyArg_ParseTuple(args, "es|es", "utf8", &emailIn, "utf8", &nameIn))
```

Comment:
https://docs.python.org/2/c-api/arg.html => "PyArg_ParseTuple() will allocate a buffer of the needed size, copy the encoded data into this buffer and adjust \*buffer to reference the newly allocated storage. **The caller is responsible for calling PyMem_Free() to free the allocated buffer after use.**"